src/components/InstrumentConfig.vue

```diff
@@ -277,17 +278,42 @@ export default {
     },
     availableOpticalElementGroups: function() {
       if (this.simpleInterface) {
-        return {
-          filters: {
-            type: 'filter',
-            label: 'Filter',
-            options: [
-              { value: 'b', text: 'Blue' },
-              { value: 'v', text: 'Green' },
-              { value: 'rp', text: 'Red' }
-            ]
-          }
-        };
+        if (this.selectedinstrument === '2M0-SCICAM-MUSCAT') {
```

Comment:
Is this block (lines 281-303) only intended to run if the simple interface is active (line 280)? If so, it looks like it is never rendered in the UI (condition in line 118) , and only used in the watch method at line 422. 

If this is true, maybe the 'type' and 'label' keys can be removed as I don't see them used anywhere.

Reply:
Good idea, will update